packages/esm-generic-patient-widgets-app/src/resources/useObs.ts

```diff
@@ -54,6 +84,18 @@ export interface UseObsResult {
 
 type ObsResult = FHIRResource['resource'] & {
   conceptUuid: string;
+  dataType: string;
+  valueString?: string;
+  valueCodeableConcept?: ObsCodedData;
```

Comment:
It would actually be better to add these to the FHIRResource type in esm-core, I think.

Reply:
@ibacher If this is added to the FHIRResource I will need to commit the esm-core since this module will depend on it.